### PR TITLE
testing/tpm2-tss-engine: new aport

### PR DESCRIPTION
**Pull request by asac**
tpm2tss engine for openssl
https://github.com/tpm2-software/tpm2-tss-engine/

### CODE REVIEW

**Comment by Cogitri** [comment]
Ah oops, forgot to actually submit my review comments after doing them :P

**Comment by asac** [comment]
all input addressed. thank you!

File: testing/tpm2-tss-engine/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributor:
```
Comment (Cogitri) [inline testing/tpm2-tss-engine/APKBUILD]: I think you forgot to write your name here :)

File: testing/tpm2-tss-engine/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Maintainer: Alexander Sack <asac@pantacor.com>
+pkgname="tpm2-tss-engine"
+pkgver="1.0.1"
+pkgrel=0
+pkgdesc="tpm2tss engine for openssl"
+url="https://github.com/tpm2-software/tpm2-tss-engine/"
+arch="all"
+license="BSD-2-Clause"
+makedepends="tpm2-tss-dev automake autoconf autoconf-archive libtool openssl-dev doxygen linux-headers curl-dev"
+source="https://github.com/tpm2-software/tpm2-tss-engine/releases/download/v1.0.1/tpm2-tss-engine-1.0.1.tar.gz"
```
Comment (Cogitri) [inline testing/tpm2-tss-engine/APKBUILD]: Don't hardcode the version number here please.

**Pull request closed without merging**